src/gnatss/loaders.py

```diff
@@ -440,3 +447,56 @@ def load_quality_control(qc_files: List[str], time_scale="tt") -> pd.DataFrame:
         qc_df = pd.DataFrame(columns=[constants.QC_STARTTIME, constants.QC_ENDTIME])
 
     return qc_df
+
+
+def read_raw_l1_data_files(
```

Comment:
```suggestion
def read_novatel_L1_data_files(
```
I think we should be a little more specific for clarity